### PR TITLE
Adds a base list of runtimes to use for testing

### PR DESCRIPTION
**Pull request by rwaldron**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -61,6 +61,18 @@ across the V8 and Test262 projects.
   $ gits push origin migration-twenty-three
   ```
 
+## Runtimes
+
+- [io.js](https://iojs.org/en/index.html)
+  + `brew install iojs`
```
Comment (jugglinmike) [inline README.md]: I don't know if it's worth including Homebrew instructions, if we don't offer instructions for any other platforms. You might say, "it couldn't hurt", but I don't want to give the impression that this is a OSX-only project.

Comment (rwaldron) [inline README.md]: I figured that we'd add other instructions for other platforms as well. I just wrote these because they are my relevant instructions.

**Comment by jugglinmike** [comment]
@rwaldron I pushed a commit to help suggest cross-platform support. Ball's in your court

**Comment by rwaldron** [comment]
Works for me

**Pull request merged**